vm/actor/src/lib.rs

```diff
@@ -40,6 +40,14 @@ impl ActorState {
             sequence,
         }
     }
+    /// Returns sequence from actor state
+    pub fn sequence(&self) -> &u64 {
+        &self.sequence
+    }
+    /// Returns balance from actor state
+    pub fn balance(&self) -> &BigUint {
+        &self.balance
+    }
```

Comment:
can just delete these since the fields are not protected (and can make state and code_id public also for consistency later but doesn't matter)